src/Amadeus/Client/RequestOptions/Service/StandaloneCatalogue/PricingOptions.php~

```diff
@@ -0,0 +1,41 @@
+<?php
```

Comment:
Remove this file please!

Reply:
If I remove this file then I need to add pricingsFareBasis property in serviceIntegratedPricingOptions.php. and pricingsFareBasis is require to implement this message.

Reply:
No, we don't want that property in a message that doesn't use it. In that case make a subclass and just add the extra property?

Reply:
done